build/bin/build-client-packages

```diff
@@ -390,6 +390,7 @@ build_rpm(){
     rm -fr rpmbuild
     mkdir -vp rpmbuild/{SOURCES,SPECS,BUILD,SRPMS,RPMS}
     cp $SRC_RPM rpmbuild/SRPMS/
+    # FIXME remove redefining pmm_version
     pmm_version=$(echo ${SRC_RPM} | awk -F'-' '{print $3}')
```

Comment:
That's what killed me – getting a version from the directory name.
Guys, we have variables! They are cool! Let's use them!